Corrected network setting checks.

GhoulofGSG9 opened the pull request:
tick rate and move rate are rather independent from each other:

The move rate sets how often per second the clients are suppose to send a move update to the server.
The tick rate set how often per second the server runs it's update tick (includes the Lua VM update tick etc).

However there is no reason to let the clients send more move packets than the server is able to process.

Also the server should not send more world updates to the clients than it has possible data avaible from the clients move update packages.

Person8880 commented on lua/shine/extensions/basecommands/server.lua:
```diff
@@ -98,8 +98,17 @@ do
+	} )
+	Validator:AddRule( {
+		Matches = function( self, Config )
+			return Config.Sendrate > Config.MoveRate
```
`SendRate` rather than `Sendrate` here and below.

Person8880 commented on lua/shine/extensions/basecommands/server.lua:
```diff
@@ -1259,10 +1268,10 @@ function Plugin:CreatePerformanceCommands()
 			NotifyError( Client, "ERROR_TICKRATE_CONSTRAINT", {
 				Rate = self.Config.MoveRate
-			}, "Tick rate cannot be greater than move rate (%i).", true, self.Config.MoveRate )
+			}, "Tick rate must be greater than move rate (%i).", true, self.Config.MoveRate )
```
Are you sure this is a must? The validator above allows them to be equal, so shouldn't this too?

Person8880 commented:
The validation was based on what you said in #369.

GhoulofGSG9 commented:
I corrected the two typos and changed the error message. I should have been more precise about what each parameter does in #369.


```Starters

    tickrate; controls how often the AI units update (default 30) and acts as an upper limit to sendrate
    sendrate; controls how often players gets updates sent to them (default 20)
    bwlimit: bandwidth-limit, how much data a player can receive per second (50kB/sec))


These interact with two other variables that has been available for some time

    mr; move rate; how often a player sends his movements and actions to the server (default is 30)
    interp; interpretation buffer; how much behind the server the client will run (default is 2/sendrate or 0.1 sec (100ms))```
